src/client/extension.ts

```diff
@@ -57,7 +59,12 @@ export async function activate(
                 !isCurrentDocumentEdited() &&
                 PortalWebView.checkDocumentIsHTML()
             ) {
-                PortalWebView?.currentPanel?._update();
+                if ( PortalWebView?.currentPanel) {
+                    const previewTelemetryData = new TelemetryData(TelemetryConstants.PORTAL_WEBPAGE_PREVIEW);
```

Comment:
Reason for creating a separate class was to provide a single/unified view of all the telemetry events/fields logged. When the project grows big and there are multiple contributors it can become difficult to get a quick view of all the telemetry that is being captured. I have removed TelemetryConstants for now since all the server data is anyways captured in a separate file TelemetryData and on the client side most likely all the events will be localized in extension.ts.